org.freedesktop.Sdk.Extension.nvidia-base.yaml

```diff
@@ -0,0 +1,104 @@
+id: org.freedesktop.Sdk.Extension.nvidia-base
+branch: '24.08'
```

Comment:
Does this need to be branched? You aren't going to maintain multiple branches for Nvidia, it will follow only one runtime version and so will this extension.

Reply:
Yeah, that makes sense. I had just copied this part from some other extension and didn't really think about it. I've removed it now.